src/analytics/helpers.js

```diff
@@ -38,9 +38,9 @@ const supportedDimensions = [
   // Dataset
   'dataset_action',
   'dataset_download_options',
-  'dataset_download_option_changes',
   'dataset_id',
   'one_off_experiment_download',
+  'regenerated_download_options_change',
```

Comment:
The prefix of this dimension name has been changed to `regenerated` to differentiate between the dataset and the regenerative dataset.